include/macros.h

```diff
@@ -26,6 +26,13 @@
 #define NOP_FIX __asm__(".set nogpopt");
 #define NOP_UNFIX __asm__(".set gpopt");
 
+#define SWAP16(in, out) \
```

Comment:
Unsure what I think of this given we only use it once in this PR but perhaps I'll change my mind if it sees use elsewhere

Reply:
I think it should go in the file its associated with, that's how its arranged in naudio.

Reply:
Sounds grood